findLongest.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * This method is to find the longest element in a Array
+ *
+ * @since 5.0.0
+ * @category Array
+ * @param {Array} array The array to inspect.
+ * @returns {string} Returns  longest element in a Array
+ * @example
+ *
+ * findLongest(["abc", 'abcd', "abcde"])
+ * // => "abcde"
+ */
+function findLongest(array) {
+  if (!Array.isArray(array)) {
+    throw new TypeError('Expected an Array')
+  }
+  if (!array.length) { return undefined }
+  const len = array.map(item => item.length)
+  const max = Math.max(...len)
```

Comment:
Shall we add tests for arrays with non-string elements?
I think this returns `NaN` if the array contains items that are not strings

Reply:
i will throw an error if arrays contain non-strings,ok?

Reply:
Should be good, yeah.

And if we are restricting to just strings, should we rename the function to something like `findLongestString`?

Reply:
year, the name is better